app/javascript/react/components/ReviewForm.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+const ReviewForm = (props) => {
+
+  let handleBodyChange = (event) => {
+    let value = event.target.value
+    props.passDownReviewChange(value);
+  }
+
+  let handleRatingChange = (event) => {
+    let value = event.target.value
+    props.passDownRatingChange(value);
+  }
+
```

Comment:
this is okay, but would probably less confusing if you do the standard practice for handleChange methods and define them in the parent container